src/public/app/core/types/ScPlayer.ts

```diff
@@ -0,0 +1,28 @@
+export interface ScPlayer {
```

Comment:
Yes. We have their sdk which has a method stream. Stream returns a promise with this player instance. This player is what actually plays the music. I figured prefixing all soundcloud provided things with `Sc`